### PR TITLE
INT-2560 Add RHCC support

### PR DESCRIPTION
**Pull request by adrianpowell**
Jira: https://issues.sonatype.org/browse/INT-2560
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/docker-nexus-iq-server-feature/job/INT-2560-add-RHCC-support/

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ARG IQ_SERVER_SHA256=81e877f4e793e740cd0be38faaa35650448bfcb77aa30fd48d15babeb01a9a09
+
+# Mandatory parameters. Docker needs to know volume mount point and location of startup script.
+ENV SONATYPE_WORK="/sonatype-work" \
+    IQ_HOME="/opt/sonatype/nexus-iq-server/" \
+    DOCKER_TYPE="rh-docker"
+
+ARG IQ_SERVER_COOKBOOK_VERSION="release-0.4.20200210-175005.e554f17"
+# ARG IQ_SERVER_COOKBOOK_URL="https://github.com/sonatype/chef-nexus-iq-server/releases/download/${IQ_SERVER_COOKBOOK_VERSION}/chef-nexus-iq-server.tar.gz"
+ARG IQ_SERVER_COOKBOOK_URL="/home/ec2-user/chef-nexus-iq-server.tar.gz"
```
Comment (fernau) [inline Dockerfile.rhel]: i think that's not right ...

**Review by fernau — CHANGES_REQUESTED** [review]

**Review by jflinchbaugh — APPROVED** [review]
+1

File: Dockerfile.rhel (hunk 1 above)
Comment (kellyrob99) [reply Dockerfile.rhel]: indeed, looks like the remote url was commented out during testing?

Comment (adrianpowell) [reply Dockerfile.rhel]: Indeed. Thanks guys - that was a testing thing (the arg wasn't getting picked up like I expected).

**Review by fernau — APPROVED** [review]
LGTM

**Pull request merged**